pathfinder/tests/test_graphs.py

```diff
@@ -27,3 +38,69 @@ def test_routing_benchmark(token_networks: List[TokenNetwork], populate_token_ne
     print(paths)
     print(np.mean(np.array(times)), np.min(np.array(times)), np.max(np.array(times)))
     print("total_runtime = {}".format(end-start))
+
+
+def test_routing_simple(
+    token_networks: List[TokenNetwork],
+    populate_token_networks_simple: None,
```

Comment:
Can we annotate this in a better way or remove the annotation.

Reply:
We could define a new `SideEffect = None` type to make it clearer that this fixture doesn't return any object but is required for the test nonetheless because of something it does to other fixtures (i.e. side effects).